README.md

```diff
@@ -1,5 +1,7 @@
 # Rechnung
 
+[![Documentation Status](https://readthedocs.org/projects/rechung/badge/?version=latest)](https://rechung.readthedocs.io/en/latest/?badge=latest)
```

Comment:
should be rechnung in the url
typo or misconfiguration?

Reply:
Unfortunately, I had a typo in the project creation. I'm not sure how to change the URL in readthedocs.

Reply:
I deleted the old project and created a new one ...